code/game/objects/structures/hivebot.dm

```diff
@@ -15,7 +15,7 @@
 	smoke.start()
 	visible_message(span_boldannounce("[src] warps in!"))
 	playsound(src.loc, 'sound/effects/empulse.ogg', 25, TRUE)
-	addtimer(CALLBACK(src, PROC_REF(warpbots)), rand(10, 600))
+	addtimer(CALLBACK(src, PROC_REF(warpbots)), rand(1, 60) SECONDS)
```

Comment:
This is changing behavior, before it could roll between-seconds and now it cannot. 
```suggestion
	addtimer(CALLBACK(src, PROC_REF(warpbots)), rand(10 SECONDS, 60 SECONDS))
```

Reply:
1 SECONDS* but i see what you mean